readabilitySAX.js

```diff
@@ -491,18 +494,18 @@ Readability.prototype.onclosetag = function(tagName){
 	else if(this._settings.cleanConditionally && tagName in cleanConditionally){
 		var p = elem.info.tagCount.p || 0,
 		    contentLength = elem.info.textLength + elem.info.linkLength;
-
+        
 		if(contentLength === 0){
 			if(elem.children.length === 0) return;
 			if(elem.children.length === 1 && typeof elem.children[0] === "string") return;
 		}
 		if((elem.info.tagCount.li - 100) > p && tagName !== "ul" && tagName !== "ol") return;
-		if(contentLength < 25 && (!("img" in elem.info.tagCount) || elem.info.tagCount.img > 2) ) return;
+		if(contentLength < 25 && ((!("img" in elem.info.tagCount) && !("iframe" in elem.info.tagCount)) || elem.info.tagCount.img > 2 || elem.info.tagCount.iframe > 2) ) return;
```

Comment:
I tested on 10-15 websites and embeds are working correct.

Reg: Safari Reader, or Firefox Reader, they do not include embeds on many readable page. But in my opinion default embed put in the normal flow of article shall be included. If they are not doing today, they will do tomorrow.

Take this page as an example -- http://www.inbound.com/blog/charlene-li-ideas-and-experiences

Embeds are in the normal flow of the article - in readable version they shall be shown, in my opinion.